customer-endpoint/src/main/kotlin/org/ostelco/prime/customer/endpoint/resources/SimProfilesResource.kt

```diff
@@ -68,6 +68,24 @@ class SimProfilesResource(private val regionCode: String, private val dao: Subsc
                         .responseBuilder()
             }.build()
 
+    @PUT
+    @Path("/{iccId}/installed")
+    @Produces(MediaType.APPLICATION_JSON)
+    fun updateSimProfile(@NotNull
+                         @PathParam("iccId")
```

Comment:
camelcase is not really the right thing to do. It's called "ICCID", so only upper or lower case is ok, camelcase not so much.

Reply:
This will change code in many places.   Will refactor in separate PR.